project/AddMetaInfLicenseFiles.scala

```diff
@@ -20,19 +20,18 @@ object AddMetaInfLicenseFiles extends AutoPlugin {
   private lazy val baseDir = LocalRootProject / baseDirectory
 
   override lazy val projectSettings = Seq(
-    apacheSonatypeLicenseFile := baseDir.value / "legal" / "StandardLicense.txt",
```

Comment:
This is no longer necessary because the default location of `apacheSonatypeLicenseFile` is `baseDir.value / "LICENSE"` which in this PR is now the default `LICENSE` file